app/views/checkout/_summary.html.haml

```diff
@@ -1,35 +1,29 @@
 %orderdetails
-  = form_for current_order, url: "#", html: {"ng-submit" => "purchase($event, checkout)"} do |f|
```

Comment:
At first this seemed odd to me, but I did a bit of research and it seems like inputs outside a `<form>` are okay:

- https://stackoverflow.com/questions/3294572/is-input-well-formed-without-a-form
- https://stackoverflow.com/questions/8568660/is-there-any-danger-to-using-input-fields-outside-without-forms-in-html-javascri